lib/rucio/common/constants.py

```diff
@@ -219,3 +219,16 @@ class RseAttr:
 
 OPENDATA_DID_STATE_LITERAL = Literal['draft', 'public', 'suspended']
 OPENDATA_DID_STATE_LITERAL_LIST = list(get_args(OPENDATA_DID_STATE_LITERAL))
+
+@enum.unique
+class HttpMethod(enum.Enum):
+    """HTTP verbs used in Rucio requests."""
+
+    # Standard HTTP/1.1
+    HEAD = 'HEAD'
+    OPTIONS = 'OPTIONS'
+    PATCH = 'PATCH'
+    GET = 'GET'
+    POST = 'POST'
+    PUT = 'PUT'
+    DELETE = 'DELETE'
```

Comment:
I would be in favour of using the enum provided by the built-in `http` library instead: https://docs.python.org/3/library/http.html#http.HTTPMethod